payment/uc_authorizenet/uc_authorizenet.install

```diff
@@ -12,13 +12,13 @@ function uc_authorizenet_requirements($phase) {
 
   $has_curl = function_exists('curl_init');
 
-  $requirements['uc_authorizenet_curl'] = array(
-    'title' => $t('cURL'),
-    'value' => $has_curl ? $t('Enabled') : $t('Not found'),
-  );
   if (!$has_curl) {
-    $requirements['uc_authorizenet_curl']['severity'] = REQUIREMENT_ERROR;
-    $requirements['uc_authorizenet_curl']['description'] = $t("Authorize.net requires the PHP <a href='!curl_url'>cURL</a> library.", array('!curl_url' => 'http://php.net/manual/en/curl.setup.php'));
+    $requirements['uc_authorizenet_curl'] = array(
+      'title' => $t('cURL'),
+      'value' => $has_curl ? $t('Enabled') : $t('cURL not found'),
```

Comment:
I find this confusing. This is inside an `if` statement that runs if `!$has_curl`, so why check for this again?

Reply:
Of course. Chalk it up to blindness.